openandroidinstaller/assets/configs/sweet.yaml

```diff
@@ -0,0 +1,57 @@
+metadata:
+  maintainer: BjornW
+  brand: xiaomi
+  device_name: Xiaomi Redmi Note 10 Pro
+  is_ab_device: false
+  device_code: sweet
+  supported_device_codes:
+    - sweet
+    - sweetin
+  untested: false
```

Comment:
Maybe also link the TWRP page here, with `twrp-link`, see [here](https://github.com/openandroidinstaller-dev/openandroidinstaller/blob/main/docs/how_to_contribute_your_own_installation_configurations.md#how-to-write-metadata)

Reply:
Good point :) This is not strictly required, but if there is a custom link necessary and the link is not just resolved by knowing the device code, then having the link is nice.

Reply:
I've added the twrp-link similar to other configs, but with this device's device code.

Reply:
You are right :+1: